terragrunt/modules/release-distribution/fastly-static.tf

```diff
@@ -115,6 +115,26 @@ resource "fastly_service_vcl" "static" {
     VCL
   }
 
+  snippet {
+    # This was an abandoned copy of Rust 1.17.0-era docs; redirect to current docs
+    name    = "redirect /doc/master to /stable"
+    type    = "error"
+    content = <<-VCL
+      if (req.url ~ "^\/doc/master(\/|$)")?$") {
```

Comment:
If I'm not counting wrong, there's a `(` missing or a `)` too many somewhere. 😅

Reply:
I tried to simplify the regex in https://github.com/rust-lang/simpleinfra/pull/571/commits/cdd0f7b54bf239773c12a60d2d45083aec4dab11